cloud/pkg/missionstatepruner/pruner.go

```diff
@@ -76,7 +76,7 @@ func (msp *MissionStatePruner) checkAndPrune() {
 	deadEdgeClusters := map[string]bool{}
 	newDeadEdgeClusters := false
 	for _, ec := range allEdgeClusters.Items {
-		if time.Since(ec.Status.LastHeartBeat.Time) > msp.edgeclusterTimeout {
+		if time.Since(ec.Status.LastHeartBeat.Time) > msp.edgeclusterTimeout || ec.Status.Healthy != "healthy" {
```

Comment:
put "healthy" string in a const variable

Reply:
done